### ISSUE TITLE
VLANs not working in macOS Catalina?

### ISSUE THREAD

**Issue by yllwfsh** [khronokernel/SmallTree-I211-AT-patch#2]
Hello all,

Just wanted to ask if anyone has the same experience with this patched SmallTree kext. I cannot seem to get VLANs to work.

I have tested this on my i211 builtin chipset and an Intel 82576 4-port server adapter. I can only connect to non-VLAN-tagged networks.

This is the result:

![image](https://user-images.githubusercontent.com/43885719/84568671-506e1700-ad81-11ea-87b4-efa001c2612e.png)

Did anyone manage to successfully connect to a VLAN using this driver and macOS Catalina? I'm on 10.15.5 (19F101) and using OC to load the kext at boot.

Thanks for your testing and reply

**Comment by tbrtje** [comment]
Hi i am experiencing the same issue. Were you able to fix it?

**Comment by yllwfsh** [comment]
No, I bought an additional PCIe network card. Good suggestions are the Realtek 8169/8111 cards for Gigabit or the Aquantia AQC-107 based cards for 10 Gbit.

**Comment by tbrtje** [comment]
Alright I thought about getting an Aquantia NIC.
I will look into this issue though and will post here if I find a solution.
